roomserver/internal/perform/perform_upgrade.go

```diff
@@ -188,7 +188,7 @@ func moveLocalAliases(ctx context.Context,
 			return fmt.Errorf("Failed to remove old room alias: %w", err)
 		}
 
-		setAliasReq := api.SetRoomAliasRequest{UserID: userID.String(), Alias: alias, RoomID: newRoomID}
+		setAliasReq := api.SetRoomAliasRequest{SenderID: senderID, Alias: alias, RoomID: newRoomID}
```

Comment:
Not an issue with this, but would you mind changing the signature of `SetRoomAlias` to `SetRoomAlias(ctx, alias, roomID, senderID)`? 
Slowly moving away from polylith function signatures, also gets rid of `SetRoomAliasRequest`and `SetRoomAliasResponse`.

Reply:
Done! And for removing aliases as well 👍